lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -5,6 +5,39 @@ module Sidekiq
     class QueueLatency
       extend Forwardable
 
+      DEFAULT_LATENCY_VALUES = {
```

Comment:
Ok, I think I'll rename to DEFAULT_QUEUE_LATENCY_VALUES. `DEFAULT_QUEUES` doesn't give any context for the values in the Hash.